libs/coin-modules/coin-cosmos/src/api/types.ts

```diff
@@ -339,3 +339,10 @@ export type PostSimulate = {
     msg_responses: { "@type": string; value: string }[];
   };
 };
+
+export type GetValidatorItem = {
```

Comment:
It would be perfect to type this fully

Reply:
Done here https://github.com/LedgerHQ/ledger-live/pull/7636/commits/edf79734ef3489a05550eca5379d074e33c1281a

Reply:
It was all strings so it was straightforward.